ros/src/computing/perception/localization/lib/gnss/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(gnss)
 
-find_package(catkin REQUIRED COMPONENTS roscpp)
+find_package(catkin REQUIRED COMPONENTS roscpp geodesy)
```

Comment:
you'll need to add an IF to don't compile this on ARM architectures, this package is not available.

Reply:
really?
I used geodesy in Jetson TX2.

Reply:
Then, please have a look at the log of GitlabCi which is failing the arm build.